app/controllers/admin/auctions/needs_attention_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Admin::Auctions::NeedsAttentionController < Admin::BaseController
+  def index
+    @view_model = Admin::NeedsAttentionAuctionsViewModel.new
```

Comment:
yeah I did not do that for the other controller but probably makes sense. If you do it, mind also renaming https://github.com/18F/micropurchase/blob/develop/app/view_models/admin/closed_auctions_list_item_view_model.rb ?